dataworkspace/dataworkspace/templates/your_files/create-table-failed.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'your_files/create-table-base.html' %}
+{% block page_title %}Table creation failed{% endblock page_title %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        We can't process your CSV file
```

Comment:
```suggestion
        We can’t process your CSV file
```

Super tiny nit, but GDS designers have sort of etched the use of "fancy apostrophes" into my brain... so might be nice to update. But optional.